### ISSUE TITLE
Fileconversion failed

### ISSUE THREAD

**Issue by L4ZUR812** [haiwen/seahub#836]
Hi,
I just set up a cloud using seafile-pro-server free version and was very happy about all going well. It is running with NginX in a non-root directory so my domain looks like "https://mydomain.de/seafile". But about two weeks later I could not view any document files but .pdf. Neither .doc or .docx nor .odt are working. I always get the message that the fileconversion failed. What I did so far:
-reinstalled libreoffice-script-provider-python and restarting seafile.sh and seahub.sh (in the following: seafile)
-disabled fileconverting, restarted seafile, enabled fileconverting, restarted seafile
-rebooted the server
-checked if everything was configured well (seahub_settings.py, ccnet.conf, seafevents.conf)

The seafevents.log says there was/is an "error invoking libreoffice: verbosity set to 3"
Well, I am not a pro when it comes to server so I do not really know whether it is a bug, but my post in the german forum was answered and the supporter said I should post it right here.
In the following is a snippet of the logfile which I think tells more about the error I am having a problem with:

[2015-10-05 20:44:27,605] [INFO] unoconv process will use python 3
[2015-10-05 20:44:27,605] [DEBUG] Running command: "/usr/bin/python3" "/home/seafile/seafile-pro-server-4.3.4/pro/python/seafevents/office_converter/unoconv$
[2015-10-05 20:44:27,613] [INFO] office converter started
[2015-10-05 20:46:35,773] [DEBUG] start to fetch task <type: doc, id: b88ab96740ef53249b9d21fb3fa28050842266ba>
[2015-10-05 20:46:37,836] [DEBUG] start to convert task <type: doc, id: b88ab96740ef53249b9d21fb3fa28050842266ba>
[2015-10-05 20:46:59,002] [WARNING] error when invoking libreoffice: Verbosity set to level 3
DEBUG: Connection type: socket,host=127.0.0.1,port=2002;urp;StarOffice.ComponentContext
DEBUG: Existing listener not found.
DEBUG: Launching our own listener using /usr/lib/libreoffice/program/soffice.bin.
Failed to connect to /usr/lib/libreoffice/program/soffice.bin (pid=4024) in 6 seconds.
Connector : couldn't connect to socket (Success)
Error: Unable to connect or start own listener. Aborting.
Using office base path: /usr/lib/libreoffice
Using office binary path: /usr/lib/libreoffice/program
LibreOffice listener successfully started. (pid=4024)

[2015-10-05 20:46:59,025] [WARNING] failed to convert <type: doc, id: b88ab96740ef53249b9d21fb3fa28050842266ba> to pdf
[2015-10-05 20:46:59,025] [DEBUG] removing temporary document /tmp/tmpaKcNCX.doc

**Comment by freeplant** [comment]
The problem will be solved in our next release for pro edition. Current restart Seafile can solve the problem.

**Comment by L4ZUR812** [comment]
Der Sir,
ich already restarted Seafile several times now and then put a monitor onto
the server and looked by  the help of the gui for the error. I got the
message that Libreoffice was not able to be started due to a failed
installation and reinstalled it. By the way I am using Libreoffice 4.4. I
reinstalled it, got the same message, after opening a specific file in the
browser to be previewed. After this, no conversion worked at all. I
researched in the internet and found out, that I have to rename the
libreoffice folder in .config to "libreoffice.old". Afterwards I converted
this specific file to .odt and all went well. So I guess it is a bug in
libreoffice and a none compatible filename (UML-.-Attribute.doc, named it
to uml_attribute.odt). Although, thank you for your request.

Sincerely,
Lars Hick
Am 13.10.2015 11:41 schrieb "Daniel Pan" <notifications@github.com>:
